mail_attach_existing_attachment/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Mail Attach Existing Attachment
+===============================
+
+This module was written to add the possibility to add attachments located on 
+the object by sending it by email with the mail compose message wizard
+
+Installation
+============
+
+To install this module, you need to:
+
+ * Click on install
+
+Configuration and Usage
+=======================
+
+To configure this module, you need to:
+
+* Adding some attachments on an object 
+
+.. figure:: static/description/attachment.png
+   :alt: Attachment on purchase order
+
+* Then, by sending the object via email, you can select the attachment added earlier
+
+.. figure:: static/description/ex_mail_compose_message.png
+   :alt: Sends the Purchase Order by email
```

Comment:
You can add the new Try me on runbot button:

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/205/8.0
```